contracts/TrueCurrencies/TrueRewardBackedToken.sol

```diff
@@ -138,10 +138,10 @@ contract TrueRewardBackedToken is RewardTokenWithReserve {
         // require TrueReward is enabled
         require(trueRewardEnabled(msg.sender), "TrueReward already disabled");
         // get balance
-        uint rewardBalance = rewardTokenBalance(msg.sender, opportunity());
+        uint256 rewardBalance = rewardTokenBalance(msg.sender, opportunity());
 
         // remove reward distribution
-        _removeDistribution(opportunity());
+        _removeDistribution();
```

Comment:
Is there a reason this function is changed? we will want to support multiple opportunities in the future. / is this relevant to this pull request